backend/dal/Repositories/LookupService.cs

```diff
@@ -54,7 +54,7 @@ public IEnumerable<PimsCountry> GetCountries()
         /// </summary>
         public IEnumerable<PimsProvinceState> GetProvinces()
         {
-            return this.Context.PimsProvinceStates.AsNoTracking().OrderBy(a => a.DisplayOrder).ThenBy(a => a.ProvinceStateCode).ToArray();
+            return this.Context.PimsProvinceStates.AsNoTracking().Where(c => c.IsDisabled != true).OrderBy(a => a.DisplayOrder).ThenBy(a => a.ProvinceStateCode).ToArray();
```

Comment:
This change should not be backported. I reverted that change in DEV in my PR for create organization